app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,13 @@
   </head>
 
   <body>
+    <% if notice %>
```

Comment:
Thanks Dan, they all work fine except the sign out feature.I managed to fix it by changing in my devise.rb file the "config.sign_out_via" from :delete to [:get,:delete] .This assures me that I have both scenarios in case the delete method doesn't work.All the changes are on a new branch, so I'm gonna close this one and make a new pull request.